recipes-extended/lucihttp/lucihttp_0.1.bb

```diff
@@ -0,0 +1,105 @@
+# that are necessarily infringed by:
+#
+# (a) their Contribution(s) (the licensed copyrights of copyright holders and
+# non-copyrightable additions of contributors, in source or binary form) alone;
+# or
+#
+# (b) combination of their Contribution(s) with the work of authorship to which
+# such Contribution(s) was added by such copyright holder or contributor, if,
+# at the time the Contribution is added, such addition causes such combination
+# to be necessarily infringed. The patent license shall not apply to any other
+# combinations which include the Contribution.
+#
+# Except as expressly stated above, no rights or licenses from any copyright
+# holder or contributor is granted under this license, whether expressly, by
+# implication, estoppel or otherwise.
+#
+# DISCLAIMER
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# NOTE
+#
+# This is part of a management middleware software package called MMX that was developed by Inango Systems Ltd.
+#
+# This version of MMX provides web and command-line management interfaces.
+#
+# Please contact us at Inango at support@inango-systems.com if you would like to hear more about
+# - other management packages, such as SNMP, TR-069 or Netconf
+# - how we can extend the data model to support all parts of your system
+# - professional sub-contract and customization services
+#
+################################################################################
+
+FILEEXTRAPATHS_prepend .= "${THISDIR}/files:"
+
+DESCRIPTION = "HTTP utility library with Lua binding"
+HOMEPAGE = "https://github.com/jow-/lucihttp"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a6b7565d075eb26cd08b6ac739db35e3"
+SECTION = "libs"
+DEPENDS = ""
+RDEPENDS_${PN} = ""
+
+PKG_MAJOR_VERSION = "0"
+
+SRC_URI = "\
+    git://github.com/jow-/lucihttp.git;protocol=https \
+    file://001-disable-werror.patch \
+    "
+SRCREV = "a34a17d501c0e23f0a91dd9d3e87697347c861ba"
+
+S = "${WORKDIR}/git"
+
+inherit cmake openwrt pkgconfig
+
+LUAPATH ?= "${libdir}/lua/5.1"
+
+PACKAGECONFIG ??= "lua"
+
+PACKAGECONFIG[lua] = "-DBUILD_LUA=ON -DLUAPATH=${LUAPATH},-DBUILD_LUA=OFF,lua5.1,lua5.1"
+PACKAGECONFIG[tests] = "-DBUILD_TESTS=ON,-DBUILD_TESTS=OFF,,"
+#OECMAKE_EXTRA = " \
```

Comment:
@MuteSpirit any reason to keep these 3 lines?
I expect tests being build after adding ```PACKAGECONFIG_append = " tests"```

Reply:
I've removed comment on line 99.

Line 97 is needed because it's used here
```
PACKAGECONFIG ??= "lua"
```

Line 98 is not needed right now but it's common practice to put possible options for PACKAGECONFIG but doesn't use in main recipe and allow do that in bbappends at external meta layers

Reply:
This recipe has been removed